spra-web/src/main/scala/net/wiringbits/spra/ui/web/components/ListGuesser.scala

```diff
@@ -49,7 +49,14 @@ object ListGuesser {
         case ColumnType.Image => Fragment()
         case ColumnType.Number => NumberInput(source = field.name)
         case ColumnType.Reference(reference, source) =>
-          defaultField(reference, field.name)(TextField(source = source))
+          ReferenceInput(
+            source = field.name,
+            reference = reference
+          )(
+            SelectInput(
```

Comment:
Let's create a ticket to consider adding an auto-complete component when the number of choices is big